src/components/Widget/components/Launcher/components/Badge/style.scss

```diff
@@ -0,0 +1,14 @@
+.launcher {
+  .badge{
+    position: fixed;
+    top: -10px;
+    right: -5px;
+    background-color: red;
```

Comment:
Set the color `red` (in hexa) in the file `variables.scss` and then use it in here